ui/packages/consul-ui/config/environment.js

```diff
@@ -176,7 +179,9 @@ module.exports = function(environment, $ = process.env) {
       ENV = Object.assign({}, ENV, {
         // in production operatorConfig is populated at consul runtime from
         // operator configuration
-        operatorConfig: {},
+        operatorConfig: {
+          APIPrefix: ''
+        },
```

Comment:
@clly would it be possible to default this to an empty string from your side? i.e. if operator hasn't set the config then it comes through as `''` to us?